examples/FactSet.AnalyticsAPI.Engines.Example/Examples/PAEngineMultipleUnitLinkedTemplatedComponentExample.cs

```diff
@@ -0,0 +1,253 @@
+            var response = linkedPATemplateApi.CreateLinkedPATemplates(linkedPATemplateParametersRoot);
+
+            var parentTemplateId = response.Data.Id;
+
+            var templatedPAComponentParametersRoot =
+                GetTemplatedPAComponentParametersRoot(paAccounts, paBenchmarks, paDates, columns, groups, parentTemplateId);
+
+            var templatedPAComponentsApi = new TemplatedPAComponentsApi(GetApiConfiguration());
+
+            var templatedPAComponentsResponse = templatedPAComponentsApi.CreateTemplatedPAComponents(templatedPAComponentParametersRoot);
+
+            var paComponentId = templatedPAComponentsResponse.Data.Id;
+
+            Console.WriteLine($"PA Component Id : {paComponentId}");
+            
+            var paCalculation = new PACalculationParameters(paComponentId, paAccounts, paBenchmarks);
+
+            var calculationParameters = new PACalculationParametersRoot
+            {
+                Data = new Dictionary<string, PACalculationParameters> {
+                    { "1", paCalculation },
+                    { "2", paCalculation }
+                }
+            };
+
+            return calculationParameters;
+        }
+
+        private static void PrintResult(ObjectRoot result)
+        {
+            Console.WriteLine("Calculation Result");
+
+            // converting the data to Package object
+            var stachBuilder = StachExtensionFactory.GetRowOrganizedBuilder();
+            var stachExtension = stachBuilder.SetPackage(result.Data).Build();
+            // To convert package to 2D tables.
+            var tables = stachExtension.ConvertToTable();
+
+            Console.WriteLine(tables[0]);
+        }
+
+        private static LinkedPATemplateParametersRoot GetLinkedPATemplateParameters(string parentComponentId)
+        {
+            var mandatory = new List<string>() { "accounts", "benchmarks" };
+            var optional = new List<string>() { "groups", "columns", "currencyisocode", "componentdetail","dates" };
+            var linkedPATemplateContent = new TemplateContentTypes(mandatory, optional);
+            var linkedPATemplateParameters = new LinkedPATemplateParameters(LinkedPATemplateDirectory,
+                parentComponentId, LinkedPATemplateDescription, linkedPATemplateContent);
+
+            var linkedPATemplateParametersRoot = new LinkedPATemplateParametersRoot(linkedPATemplateParameters);
+            return linkedPATemplateParametersRoot;
+        }
+
+        private static TemplatedPAComponentParametersRoot GetTemplatedPAComponentParametersRoot(
+            List<PAIdentifier> paAccounts, List<PAIdentifier> paBenchmarks,
+            PADateParameters paDates, List<PACalculationColumn> columns, List<PACalculationGroup> groups, string parentTemplateId)
+        {
+            var templatedPAComponentData =
+                new PAComponentData(paAccounts, paBenchmarks, groups, columns, paDates, CurrencyISOCode, ComponentDetail);
```

Comment:
Let's not use a constructor. Can you please use set methods? With new properties, the order of arguments in the constructor can change. Please make this change for both python and dotnet PRs.

Reply:
fixed https://github.com/factset/analyticsapi-engines-dotnet-sdk/pull/68/commits/f76c9f2dc71e65ea5e25734ca8d408f49ffe4e75